app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -64,3 +64,7 @@
 .gem-c-document-list--top-margin {
   margin-top: $gutter-two-thirds;
 }
+
+.gem-c-document-list__multi-list {
+  width: 100%;
```

Comment:
I've just noticed that the right edge of this (the grey dividing border) seems to overlap the page width (although I'm not sure how this will look in production). See below.

<img width="830" alt="screen shot 2018-08-10 at 16 41 09" src="https://user-images.githubusercontent.com/861310/43967445-52231640-9cbc-11e8-9185-3154dabc41ff.png">

Might need to put a margin-right on it of 25px to counter the negative margin on the parent.

Reply:
Good spot, I thought it didn't look quite right! 👍